SmartDeviceLink/public/SDLFunctionID.m

```diff
@@ -117,7 +117,8 @@ - (instancetype)init {
                          @32785: SDLRPCFunctionNameOnRCStatus,
                          @32786: SDLRPCFunctionNameOnAppServiceData,
                          @32787: SDLRPCFunctionNameOnSystemCapabilityUpdated,
-                         @32788: SDLRPCFunctionNameOnSubtleAlertPressed,
+//                         @32788: SDLRPCFunctionNameOnSubtleAlertPressed,  //Warning: duplicate f-id
```

Comment:
if this is not used, we can remove it

Reply:
<element name="OnAppCapabilityUpdatedID" value="32791" hexvalue="8017" since="7.1"/>
The function id had a conflict with another one, looks like it is resolved in MOBILE_API.xml